src/action.js

```diff
@@ -84,11 +89,13 @@ function markdownReport(report, commit, options) {
     const fileTotal = Math.round(file.total);
     const fileLines = Math.round(file.line);
     const fileBranch = Math.round(file.branch);
+    const fileMissing = file.missing;
     files.push([
       showClassNames ? file.name : file.filename,
       `\`${fileTotal}%\``,
       showLine ? `\`${fileLines}%\`` : undefined,
       showBranch ? `\`${fileBranch}%\`` : undefined,
+      showMissing ? `\`${fileMissing}\`` : undefined,
```

Comment:
Actually, might be a bit more clever to put this after the `status` icon. As we might get quite a skewed table otherwise.

Could also be worth looking in to cutting the string off(e.g. `...`), when it's long(?)

Reply:
Sorry for the late review..

> Actually, might be a bit more clever to put this after the status icon. As we might get quite a skewed table otherwise.

I agree, let's put it after the icon!

> Could also be worth looking in to cutting the string off(e.g. ...), when it's long(?)

Good idea, I think this should be configurable by an option.

Reply:
I included another option `show_missing_crop`. Tried to signal that it's for configuring `show_missing` by having `show_missing` as prefix. Might not be the best name here, suggestions are welcome.

I made it so that it expects an integer, symboling the position of a crop. Where also "..." will be appended when a crop has happened.

Reply:
Maybe something `show_missing_max_length` is more descriptive about the value. What do you think?

Reply:
That has a better ring to it. I'll stick to that.